files/build_templates/sonic_debian_extension.j2

```diff
@@ -269,6 +269,9 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-ztp_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 {% endif %}
 
+sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/libdashapi_*.deb || \
```

Comment:
Is python3-protobuf already installed on the host?

Reply:
protobuf is installed in the sonic-utilities PR:
https://github.com/sonic-net/sonic-utilities/blob/1cca178572f82fd88d2e9cedeb707e997e3a1480/setup.py#L240